src/components/AppNavigationItem/AppNavigationNewItem.vue

```diff
@@ -0,0 +1,160 @@
+  <!--
+  - @copyright Copyright (c) 2020 Jonathan Treffler <mail@jonathan-treffler.de>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
```

Comment:
Indent is off.
You're also missing the author.

You can copy the header from any other vue file we have here :)

Reply:
Who is the author ? Is it per file ? I have no idea how that stuff works

Reply:
You are :)

Reply:
Yes it is per file, you can also add yourself to the other files you edited (on the author only, not the @copyright though)

Reply:
I think its fixed now